templates/zerver/help/import-from-gitter.md

```diff
@@ -109,9 +130,29 @@ to mark the appropriate users as organization owners.
   organization using [this
   tool](https://github.com/minrk/archive-gitter/pull/5).
 
-- This tool doesn't translate Gitter's Markdown format into Zulip
-  format Markdown (there are a few corner cases where the syntax is
-  different).  Additionally, Gitter's issue mention syntax isn't translated.
+- This tool doesn't translate Gitter's Markdown format into Zulip's Markdown
+  format (there are a few corner cases where the syntax is different).
+  Additionally, Gitter's issue mention syntax isn't translated.
+
+- Messages in threads are imported, but they are not explicitly marked as
+  being in a thread.
```

Comment:
Not sure if we have enough confidence to say this here? If I understand Gitter export/import correctly, the export is done by an unofficial tool https://github.com/minrk/archive-gitter which may or may not fetch messages in thread at all - like e.g. according to this doc, it doesn't fetch private channels. So maybe we shouldn't mention this and just open an issue for someone maybe manually testing how this actually is?

ACTUALLY, on another note - perhaps private channels (I think that's the same as rooms? https://blog.gitter.im/2014/03/14/introducing-channels/)  are able to be exported and we should delete that mention? See https://github.com/minrk/archive-gitter/commit/a21abd800a496001a632c613de61fd1a491851f6 - funnily that was merged basically at the same time as our 104b2342d94ef7406bc30a1928d34be147128caa documenting the alleged unexportability :thinking:

Reply:
Yeah, good point. Also it looks like threaded conversations were [announced](https://blog.gitter.im/2020/05/04/threaded-conversations-and-gitlab-based-communities-and-rooms) after the unofficial export tool was last updated so it's likely that they simply get ignored in the export process. I think we should drop both bullet points.